scalars.go

```diff
@@ -153,6 +191,9 @@ func coerceFloat(value interface{}) interface{} {
 	case *string:
 		return coerceFloat(*value)
 	}
+
+	// TODO: check the graphql spec: coerceInt returns nil if it
```

Comment:
- Yes, we should return `nil`.
- Addressed by :https://github.com/graphql-go/graphql/pull/352